docs/vm-configuration.md

```diff
@@ -0,0 +1,265 @@
+Virtual Machine Configuration
+=============================
+
+The configuration of virtual machines is one of the core tasks to accomplish
+with kubevirt. At the public kubernetes API layer, this will involve the user
+creating manifests that follow the published schema for one or more third party
```

Comment:
There's a new name to 'third party resources' in K8S now. Forgot what it is though.